eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -139,16 +110,10 @@ async function getTasks() {
 
 // function to create the entire calander
 async function createCal() {
-    console.log("creating calander...");
-    // const response = await fetch("/calendar");
-    // const text = await response.json().then(data =>{
-    // console.log(data);
-    // });
+    //Gets scheduled tasks
     var tasks = await fetch('/calendar').then(res => res.json());
-    console.log(tasks);
-      
     var CLIENT_ID = '470404283189-q3gbv28dhmra1bg82g1evcn4c6gt3d2k.apps.googleusercontent.com';
-    var API_KEY = 'AIzaSyBtsuyHcg_Ei9wf2bdx7IZ-DdY56CnY3jU';
+    var API_KEY = 'AIzaSyAEXSdXQrLWuCEGsP1Usxi_lS4GTXmRRKA';
```

Comment:
Just curious, if you already check the environment variables? that way we won't expose the keys.

Reply:
I was looking into the environment variables, but when I asked Kevin Workman he said that as long as you're restricting the API keys you can expose the keys like this. I was definitely going for something working though and do agree its an issue.